<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaPlayerPD_321</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/ico/ico/sound_icon-icons.com_74449.ico</normaloff>:/ico/ico/sound_icon-icons.com_74449.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPause">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderProgress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderVolume">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelVolume">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDuration">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>79</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProgress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress:</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewPlaylist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>571</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMute">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLoop">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Loop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxShuffle">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Shuffle</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBitrate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>bitrate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSampleRate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample rate:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClr">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CLR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
